src/__tests__/index.js

```diff
@@ -148,7 +148,7 @@ import Demo from './demo'
   assert.equal(
     error.message,
     `Build failed with 1 error:
-__mdx_bundler_fake_dir__${path.sep}index.mdx:2:17: error: [inMemory] Could not resolve "./demo" from the entry MDX file.`,
+__mdx_bundler_fake_dir__/index.mdx:2:17: error: [inMemory] Could not resolve "./demo" from the entry MDX file.`,
```

Comment:
Not sure why this was changed:

```suggestion
__mdx_bundler_fake_dir__${path.sep}index.mdx:2:17: error: [inMemory] Could not resolve "./demo" from the entry MDX file.`,
```

Otherwise the build will break on windows machines, won't it?

Reply:
It was breaking builds on my windows machine.

Node is really inconsistent on when it uses the windows seperator or not. Because we supplied `__mdx_bundler_fake_dir__/index.mdx` to esbuild thats the error we get. We only need `path.sep` when dealing with a returned file path. like on line 171.